packages/inventory-compliance/src/SystemRulesTable.test.js

```diff
@@ -89,20 +92,83 @@ describe('SystemRulesTable component', () => {
                 profileRules={ profileRules }
                 loading={ false }
                 system={ system }
+                itemsPerPage={ 100 }
             />
         );
         const instance = wrapper.instance();
         await instance.setInitialCurrentRows();
-        expect(wrapper.state('currentRows').length / 2).toEqual(2);
+        expect(wrapper.state('currentRows').length / 2).toEqual(52);
         expect(wrapper.state('currentRows')[0].cells[TITLE_COLUMN].original).
         toEqual('Use direct-lvm with the Device Mapper Storage Driver');
         expect(wrapper.state('currentRows')[2].cells[TITLE_COLUMN].original).
         toEqual('Enable the Docker service');
         await instance.onSort(null, TITLE_COLUMN + 2, SortByDirection.asc);
-        expect(wrapper.state('currentRows').length / 2).toEqual(2);
+        expect(wrapper.state('currentRows').length / 2).toEqual(52);
         expect(wrapper.state('currentRows')[0].cells[TITLE_COLUMN].original).
-        toEqual('Enable the Docker service');
+        toEqual('Add nodev Option to /dev/shm');
```

Comment:
After adding more results to the response, the first alphabetical row title has changed